tests/Avalonia.Base.UnitTests/Data/DefaultValueConverterTests.cs

```diff
@@ -251,6 +250,11 @@ public override bool Equals(object obj)
             {
                 return obj is CustomType other && this.Value == other.Value;
             }
+
+            public override int GetHashCode()
+            {
+                return HashCode.Combine(Value);
```

Comment:
HashCode.Combine do not work on Netstandard2.0. Can i retarget on netstandard2.1?